src/transport/MessageHeader.cpp

```diff
@@ -30,13 +30,21 @@
 /**********************************************
  * Header format (little endian):
  *
- *  16 bit: | VERSION: 4 bit | FLAGS: 4 bit | RESERVED: 8 bit |
- *  16 bit: | Secure message type                             |
- *  32 bit: | MESSAGE_ID                                      |
- *  32 bit: | Secure Session ID                               |
- *  64 bit: | Message Authentication Tag                      |
- *  64 bit: | SOURCE_NODE_ID (iff source node flag is set)    |
- *  64 bit: | DEST_NODE_ID (iff destination node flag is set) |
+ * -------- Unencrypted header -----------------------------------------------------
+ *  16 bit: | VERSION: 4 bit | FLAGS: 4 bit | ENCRYPTTYPE: 4 bit | RESERVED: 4 bit |
+ *  32 bit: | MESSAGE_ID                                                           |
+ *  64 bit: | SOURCE_NODE_ID (iff source node flag is set)                         |
+ *  64 bit: | DEST_NODE_ID (iff destination node flag is set)                      |
+ *  16 bit: | Encryption Key ID                                                    |
+ *  16 bit: | Payload Length                                                       |
+ * -------- Encrypted header -------------------------------------------------------
+ *  8 bit:  | Exchange Header                                                      |
+ *  8 bit:  | Exchange Message Type                                                |
```

Comment:
Exchange Message type seems to be a new field previously undefined by the Weave Message Layer. What is this and how is it to be used?

Reply:
@gerickson this issue (https://github.com/project-chip/connectedhomeip/issues/1692) defines the field.

Reply:
Changed it to MessageType.

Reply:
It sounds like from talking to @turon, this was a simple typo. Thanks for the correction and clarification.